api/RTCIceCandidateInit.json

```diff
@@ -67,10 +67,12 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": "22"
+              "version_added": "22",
+              "notes": "In Firefox 68, the <code>candidate</code> property became optional per mid-2017 specification update."
```

Comment:
OK, I'm worried I might be indulging in some perfectionism here, but there's two things I noticed in your revised notes:

1. I'd prefer to have some consistency across the notes in the PR. I prefer the backward-looking "Before Firefox 68" for a spec change like this (which frames the old behavior as noteworthy, not the current behavior). I'd like to see this throughout, but if you think this is too nitpicky, that's OK, I can move on. 😄 

2. For the pair of notes in this file, I think this new phrasing, "In Firefox 68", is ambiguous. It might be read as as exceptional, as if to mean "In Firefox 68 and only Firefox 68", which I don't think is what you intended. Depending on how you feel about point 1, I'd like to see this change to either "Before Firefox 68" (backward-looking) or "Starting in Firefox 68" (forward-looking).

Reply:
Well, the construct "In Firefox 68 <blah blah> became optional" indicates a change that took place and that has not been undone. If it were "In Firefox 68 <blah blah> is optional" that would be very different. Personally I see no need to change any of this but I'll take a look in a few minutes.

Reply:
As for consistency... I think variety helps with the experience of reading content. Seeing variations on the same phrase over and over again becomes tedious and causes the eye to begin to glaze over and miss things.